src/main/java/com/stuypulse/robot/commands/DrivetrainCommand.java

```diff
@@ -30,7 +31,7 @@ public DrivetrainCommand(Drivetrain drivetrain) {
 
     public abstract double getAngle();
 
-    public abstract Drivetrain.Gear getGear();
+    public abstract Gear getGear();
```

Comment:
I am not a huge fan of the Gear class not being a part of Drivetrain.

Reply:
I can move it back!